lib/msf/core/auxiliary/mixins.rb

```diff
@@ -8,6 +8,7 @@
 require 'msf/core/auxiliary/drdos'
 require 'msf/core/auxiliary/fuzzer'
 require 'msf/core/auxiliary/report'
+require 'msf/core/auxiliary/recog'
```

Comment:
These requires appear to previously have been ordered alphabetically, so please put recog above report so later developers can spot if a file is already required by ordered scanning.